validation-test/stdlib/UnsafeBufferPointer.swift.gyb

```diff
@@ -15,6 +15,8 @@ struct SubscriptGetTest {
   /// The values that should be expected by slicing the UBP, or `nil` if the
   /// test is expected to crash.
   let expectedValues: [Int]?
+  /// Same as `expectedValues`, but for closed ranges. `nil` if no difference.
+  let expectedClosedValues: [Int]?
```

Comment:
Those tests don't seem to be instrumented for use with the different types of ranges. Would a follow-up item to fix that be of interest to you?